server/server/src/cold_chain/temperature_log.rs

```diff
@@ -7,7 +7,6 @@ use anyhow::Context;
 use chrono::NaiveDateTime;
 use log::error;
 use mime_guess::mime;
-use openssl::error;
```

Comment:
This unused import was annoying me, but is side track...

Reply:
thanks - I was going to take that out too!